include/dsn/dist/replication/meta_service_app.h

```diff
@@ -59,7 +59,6 @@ class meta_service_app : public service_app
 private:
     friend class ::dsn::replication::test::test_checker;
     std::unique_ptr<dsn::replication::meta_service> _service;
-    version_http_service *_version_http_service;
```

Comment:
I see that this pull request remove `version_http_service`, is it useless?

Reply:
No. I will create `version_http_service` in pegasus side. I've included "Known issue" section on my description.